bill/discounts.go

```diff
@@ -67,7 +67,7 @@ func (m *Discount) Validate() error {
 }
 
 // GetTaxRates responds with the array of tax rates applied to this line.
-func (m *Discount) GetTaxRates() tax.Rates {
+func (m *Discount) GetTaxes() tax.Map {
```

Comment:
Stylistic: Go convention is to omit the word 'Get' from getters.

Also: Why do we need getters for exported fields?  Maybe to satisfy an interface?